functions/chain.compile.fish

```diff
@@ -15,7 +22,7 @@ function chain.compile -d 'Compiles the prompt'
             and echo -n '-'
             or set next
 
-          builtin printf '<%s>' \"\$segment[2]\"
+          builtin printf '"$chain_link_open_glyph%s$chain_link_close_glyph"' \"\$segment[2]\"
```

Comment:
Should the `%s` be in quotes? I don't know if any versions of Fish have `%` as a special symbol. Just in case, I'd change it to

```
builtin printf '"$chain_link_open_glyph"%s"$chain_link_close_glyph"' \"\$segment[2]\"
```

Reply:
In that case wouldn't just `builtin printf '$chain_link_open_glyph%s$chain_link_close_glyph' \"\$segment[2]\"` work, too? I.e. leaving away the double quotes entirely? Sorry, I'm a fish noob 😛